.github/workflows/release.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           DIST_TAGS_URL="https://api.github.com/repos/${{ github.repository }}/tags"
           DIST_TAG_FOUND=`curl -s ${DIST_TAGS_URL} --connect-timeout 5 | grep "\"name\"\: \"${CORE_LATEST_VER}\""` || true
           if [ -z ${DIST_TAG_FOUND} ]; then
-            rm -rf *.zip *.dgst
+            find . -type 'f' | grep -v "README.md" | grep -v ".git" | grep -v ".github" | xargs rm -rf
```

Comment:
A small problem:
`.git` and `.github` are directories, not files, so
```
grep -v ".git" | grep -v ".github"
```
does not make sense.

Reply:
Files in `.git` and `.github` will be removed if not.

Reply:
I tested it in macOS `find`. It will list files in sub-directories too.

Reply:
I will use `find` to exclude paths in a new PR.